integration/tests/cook/test_basic.py

```diff
@@ -869,6 +869,94 @@ def query():
             # (ensure it still works when there are no live jobs)
             util.kill_groups(self.cook_url, [group_uuid])
 
+    def test_group_change_killed_retries(self):
+        group_spec = self.minimal_group()
+        group_uuid = group_spec['uuid']
+        job_spec = {'group': group_uuid, 'command': f'sleep 1'}
+        try:
+            jobs, resp = util.submit_jobs(self.cook_url, job_spec, 10, groups=[group_spec])
+            self.assertEqual(resp.status_code, 201)
+            # wait for some job to start
```

Comment:
Nitpick: misplaced comment?

Reply:
This definitely looks out of place after adding the blank lines around he def. I moved it down to just before the corresponding `wait_until` call.